src/components/Shared/DatePicker.vue

```diff
@@ -113,7 +113,7 @@ export default {
 				days: getDayNamesShort(),
 				months: getMonthNamesShort(),
 				placeholder: {
-					date: t('calendar', 'Select Date'),
+					date: this.$t('calendar', 'Select Date'),
```

Comment:
Why isn't `t` / `translate` imported here?

Reply:
because we are inside a Vue component and in main.js, we set `Vue.prototype.$t = translate`